From 2c3ba45bedac6b04dfa606926f59ebf1db72769e Mon Sep 17 00:00:00 2001
From: Dheerendra Purohit <dheerendra@pthinks.com>
Date: Mon, 14 Jul 2025 15:59:07 +0530
Subject: [PATCH 410/553] Doc: Clarify usage of QTextFormat::FontSizeAdjustment

Specify that the value is an integer and is added to the base font size.

Pick-to: 6.9
Fixes: QTBUG-130805
Change-Id: I4093fed2412cf009049fa28152a0d71ee2f81844
Reviewed-by: Leena Miettinen <riitta-leena.miettinen@qt.io>
Reviewed-by: Marcus Tillmanns <marcus.tillmanns@qt.io>
(cherry picked from commit 91d93a739d11b64dd66fe1aeb0bf469544403322)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/gui/text/qtextformat.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gui/text/qtextformat.cpp b/src/gui/text/qtextformat.cpp
index a4232ecaff7..c003a26448b 100644
--- a/src/gui/text/qtextformat.cpp
+++ b/src/gui/text/qtextformat.cpp
@@ -630,8 +630,8 @@ Q_GUI_EXPORT QDataStream &operator>>(QDataStream &stream, QTextTableCellFormat &
     \value FontStyleName
     \value FontPointSize
     \value FontPixelSize
-    \value FontSizeAdjustment       Specifies the change in size given to the fontsize already set using
-                                    FontPointSize or FontPixelSize.
+    \value FontSizeAdjustment       Specifies an integer adjustment added to the base font size set using
+                                    \c FontPointSize or \c FontPixelSize.
     \value FontFixedPitch
     \omitvalue FontSizeIncrement
     \value FontWeight
-- 
2.51.2

